src/footer/js/footer-language.js

```diff
@@ -65,7 +64,7 @@ export default (data, prefix) => {
 	});
 
 	$barrier.addEventListener('footer:update:language', ({detail}) => {
-		$control.innerHTML = detail.buttonLabel;
```

Comment:
is this meant to be hard-coded?

Reply:
@maccesar21 please change this back to use the passed in value